man/InitDeviceTcti.3.in

```diff
@@ -1,26 +1,21 @@
 .\" Process this file with
```

Comment:
I think you should also rename the file accordingly...

However, I'd wait for merge of #808 and rebase on top of that to prevent conflicts in the Makefile...

Reply:
yes of course

Reply:
@AndreasFuchsSIT : I just merged #808  & addressed comments here.